.github/workflows/import/index.js

```diff
@@ -118,8 +118,8 @@ async function importUrl(url) {
   console.log("Started path: ", url.href);
   const [fromRepo, fromOrg] = url.hostname.split('.')[0].split('--').slice(1).slice(-2);
   if (!(fromRepo || fromOrg)) {
-    console.log(liveDomain, url.origin.startsWith(liveDomain));
-    if (!(liveDomain && url.origin.startsWith(liveDomain))) {
+    console.log(liveDomain, url.origin === liveDomain);
+    if (url.origin !== liveDomain) {
```

Comment:
This was originally from https://github.com/da-sites/nexter/blob/main/nx/blocks/importer/index.js#L100 and I tried to keep it 1:1 as the source where possible, unless the env diff (browser|nodejs) forces us to ... from given this turns more into a bit of a fork with the same core functionality should be fine.